src/components/CourseOutlineStatus/index.jsx

```diff
@@ -159,28 +224,20 @@ export default class CourseOutlineStatus extends React.Component {
     );
   }
 
-  getBody = () => (
-    this.isLoading() ?
-      this.getLoadingIcon() :
-      this.getHyperLink()
-  );
-
   isLoading = () => (
     this.props.loadingChecklists.includes(checklistLoading.COURSE_BEST_PRACTICES) ||
     this.props.loadingChecklists.includes(checklistLoading.COURSE_LAUNCH)
   );
+  doesCourseHaveReleaseDate = () => (this.props.studioDetails.course.course_release_date !== 'Set Date');
 
   render() {
     return (
       <React.Fragment>
-        <div className={styles['status-checklist']}>
-          <h2 className={styles['status-checklist-label']}>
-            <WrappedMessage message={messages.checklistLabel} />
-          </h2>
-          <div>
-            {this.getAriaLiveRegion()}
-            {this.getBody()}
-          </div>
+        {this.getAriaLiveRegion()}
+        <div className="row no-gutters">
+          {this.getCourseStartDateStatus()}
+          {this.getCoursePacingTypeStatus()}
+          {this.getCourseChecklistStatus()}
```

Comment:
I know this won't always be the correct thing to do, but something about the ease of reading what gets rendered in this render function feels really good.

Reply:
Yay! :)